sha3-asm/build.rs

```diff
@@ -18,7 +18,24 @@ fn main() {
     if target.is_any_arm() {
         cc.include("cryptogams/arm");
     }
-    cc.file(sha3).compile("keccak");
+    cc.file(sha3)
+        .flag("-D")
```

Comment:
should make this programmatic with an array/forloop, also needs /D instead of -D for MSVC